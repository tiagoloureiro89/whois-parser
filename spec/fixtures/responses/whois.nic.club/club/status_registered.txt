Domain Name: google.club
Registry Domain ID: D4969-CLUB
Registrar WHOIS Server:
Registrar URL: www.markmonitor.com
Updated Date: 2019-02-14T19:55:59Z
Creation Date: 2014-03-29T15:15:13Z
Registry Expiry Date: 2019-03-28T23:59:59Z
Registrar: MarkMonitor, Inc.
Registrar IANA ID: 292
Registrar Abuse Contact Email: abusecomplaints@markmonitor.com
Registrar Abuse Contact Phone: +1.2083895740
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Domain Status: inactive https://icann.org/epp#inactive
Registry Registrant ID:
Registrant Name:
Registrant Organization: Google LLC
Registrant Street:
Registrant Street:
Registrant Street:
Registrant City:
Registrant State/Province: CA
Registrant Postal Code:
Registrant Country: US
Registrant Phone:
Registrant Phone Ext:
Registrant Fax:
Registrant Fax Ext:
Registrant Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Registry Admin ID:
Admin Name:
Admin Organization:
Admin Street:
Admin Street:
Admin Street:
Admin City:
Admin State/Province:
Admin Postal Code:
Admin Country:
Admin Phone:
Admin Phone Ext:
Admin Fax:
Admin Fax Ext:
Admin Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Registry Tech ID:
Tech Name:
Tech Organization:
Tech Street:
Tech Street:
Tech Street:
Tech City:
Tech State/Province:
Tech Postal Code:
Tech Country:
Tech Phone:
Tech Phone Ext:
Tech Fax:
Tech Fax Ext:
Tech Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Name Server:
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2019-02-20T12:47:42Z <<<