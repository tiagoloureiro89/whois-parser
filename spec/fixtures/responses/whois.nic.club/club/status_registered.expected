#domain
  %s == "google.club"

#domain_id
  %s == "D4969-CLUB"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2014-03-29T15:15:13Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2019-02-14T19:55:59Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2019-03-28T23:59:59Z}


#registrar
  %s %CLASS{registrar}
  %s.id           == "292"
  %s.name         == "MarkMonitor, Inc."

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == ""
  %s[0].name          == ""
  %s[0].organization  == "Google LLC"
  %s[0].address       == ""
  %s[0].city          == ""
  %s[0].zip           == ""
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == ""
  %s[0].fax           == ""
  %s[0].email         == ""

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == ""
  %s[0].name          == ""
  %s[0].organization  == ""
  %s[0].address       == ""
  %s[0].city          == ""
  %s[0].zip           == ""
  %s[0].state         == ""
  %s[0].country       == nil
  %s[0].country_code  == ""
  %s[0].phone         == ""
  %s[0].fax           == ""
  %s[0].email         == ""

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == ""
  %s[0].name          == ""
  %s[0].organization  == ""
  %s[0].address       == ""
  %s[0].city          == ""
  %s[0].zip           == ""
  %s[0].state         == ""
  %s[0].country       == nil
  %s[0].country_code  == ""
  %s[0].phone         == ""
  %s[0].fax           == ""
  %s[0].email         == ""


#nameservers - unsigned DNSSEC @ Feb 2019
#  %s %CLASS{array}
#  %s %SIZE{4}
#  %s[0] %CLASS{nameserver}
#  %s[0].name == "ns1.google.com"
#  %s[1] %CLASS{nameserver}
#  %s[1].name == "ns2.google.com"
#  %s[2] %CLASS{nameserver}
#  %s[2].name == "ns4.google.com"
#  %s[3] %CLASS{nameserver}
#  %s[3].name == "ns3.google.com"
